website/docs/docs/dbt-cloud-apis/discovery-querying.md

```diff
@@ -64,6 +62,18 @@ Every query will require an environment ID or job ID. You can get the ID from a
 There are several illustrative example queries on this page. For more examples, refer to [Use cases and examples for the Discovery API](/docs/dbt-cloud-apis/discovery-use-cases-and-examples).
 
 
+## Discovery API endpoints
+
+The following are the endpoints for accessing the Discovery API. Use the one that's appropriate for your region and plan.
+
+| Deployment type |	Discovery API URL |
+| --------------- | ------------------- |
+| North America multi-tenant	|	https://metadata.cloud.getdbt.com/graphql |
+| EMEA multi-tenant	|	https://metadata.emea.dbt.com/graphql |
+| APAC multi-tenant	|	https://metadata.au.dbt.com/graphql |
+| Multi-cell	| `https://YOUR_ACCOUNT_PREFIX.metadata.REGION.dbt.com/graphql`<br /><br />  Replace `YOUR_ACCOUNT_PREFIX` with your specific account identifier and `REGION` with your location, which could be `us1.dbt.com`. |<br />
+| Single-tenant | `https://metadata.YOUR_ACCESS_URL/graphql`<br /><br />  Replace `YOUR_ACCESS_URL` with your specific account prefix with the appropriate [Access URL](/docs/cloud/about-cloud/access-regions-ip-addresses) for your region and plan.|
```

Comment:
does this need the `/api` in the ST URL?
```suggestion
| Single-tenant | `https://metadata.YOUR_ACCESS_URL/api/graphql`<br /><br />  Replace `YOUR_ACCESS_URL` with your specific account prefix with the appropriate [Access URL](/docs/cloud/about-cloud/access-regions-ip-addresses) for your region and plan.|
```

Reply:
@eddowh : wdyt?

Reply:
nope it doesn't. see Ally for example: https://metadata.cloud.ally.getdbt.com/graphql